dashboard/app/models/levels/ailab.rb

```diff
@@ -45,16 +45,45 @@ def self.create_from_level_builder(params, level_params)
     )
   end
 
+  def localized_dynamic_instructions
+    default_value = JSONValue.value(properties['dynamic_instructions'].presence)
+    return default_value if default_value.nil?
+
+    parsed_default_value = JSON.parse(default_value)
+    parsed_default_value.keys.each do |prop|
+      translated_prop = I18n.t(
+        prop,
+        scope: [:data, 'dynamic_instructions', name],
+        default: nil,
+        smart: true
+      )
+      parsed_default_value[prop] = translated_prop unless translated_prop.nil?
```

Comment:
Just curious if this is the same as just assigning it and using a `default: parsed_default_value[prop]` which I think is similar to how it is done in other places.

Reply:
Good catch I will update this